bin/commands/disable.js

```diff
@@ -3,13 +3,13 @@
 var
   rc = require('rc'),
   params = rc('kuzzle'),
-  Kuzzle = require('../../lib/api');
+  KuzzleServer = require('../../lib/api/kuzzle');
```

Comment:
should we rather reuse "Kuzzle" term instead of "KuzzleServer" ?
(same remark for all other commands)